docs/mkdocs.yml

```diff
@@ -85,8 +85,8 @@ nav:
   - Triage documentation:
       - Experiment:
           - Defining an Experiment: experiments/defining.md
+          - Experiment Configuration: experiments/experiment-config.md
```

Comment:
I think you may want to have this replace defining.md. defining.md was created as a placeholder for the type of document you have now in experiment-config.md

Reply:
I was wondering what the plans for that document were - this makes sense.